example/test_calendar.yaml

```diff
@@ -23,3 +37,16 @@ events:
         # seconds, minutes, hours, days, weeks, months, years
         years: 1
       until: 2030-04-22
+
+  # We can use a nice trick of YAML (anchors and references) to make a
+  # series of events that have the same template and only a small
+  # variation.
+  # The & is the anchor: it identifies this mapping as 'template-name'
+  - &template-name
+    summary: Template event
+    begin: 2022-02-21 14:00:00
+    duration: {minutes: 60}
+  # This uses the 'template-name' template, and then overrides certain
+  # fields: in this case, the 'begin'.
+  - <<: *template-name
+    begin: 2022-02-28 14:00:00
```

Comment:
Maybe this can go in a different place, like an advanced tips and tricks in the docs (README, for now); I find that syntax very confusing, and I imagine newcomers do too.

Reply:
We might as well remove this completely. This is "just" yaml syntax and out of scope here.

Reply:
Yet super useful to avoid copy-and-paste (and not very well known).  I was going to move it to another advanced_example.yaml calendar in a separate PR (since the original PR was now pulled into here - was that necessary).

Since I already know of this, I'm of course fine if it is removed.

Reply:
OK, let's remove for now.